firebase-firestore/src/main/java/com/google/firebase/firestore/Query.java

```diff
@@ -278,10 +301,29 @@ Query whereArrayContainsAny(@NonNull String field, @NonNull List<Object> value)
    */
   // TODO(in-queries): Expose to public once backend is ready.
   @NonNull
-  Query whereArrayContainsAny(@NonNull FieldPath fieldPath, @NonNull List<Object> value) {
+  Query whereArrayContainsAny(@NonNull FieldPath fieldPath, @NonNull List<? extends Object> value) {
     return whereHelper(fieldPath, Operator.ARRAY_CONTAINS_ANY, value);
   }
 
+  /**
+   * Creates and returns a new {@code Query} with the additional filter that documents must contain
+   * the specified field, the value must be an array, and that the array must contain at least one
+   * value from the provided array.
+   *
+   * <p>A {@code Query} can have only one {@code whereArrayContainsAny()} filter and it cannot be
+   * combined with {@code whereArrayContains()} or {@code whereIn()}.
+   *
+   * @param fieldPath The path of the field containing an array to search.
+   * @param first The first value of the array that contains the values to match.
+   * @param rest The other values of the array that contain the values to match.
+   * @return The created {@code Query}.
+   */
+  // TODO(in-queries): Expose to public once backend is ready.
+  @NonNull
+  Query whereArrayContainsAny(@NonNull FieldPath fieldPath, Object first, Object... rest) {
```

Comment:
Maybe rename first argument to "value" and "rest" to "moreValues" (which matches our update() varargs method).